www/activities/settings/views/diary.html

```diff
@@ -39,6 +39,20 @@
           </a>
         </li>
 
+        <li>
+          <div class="item-content">
+            <div class="item-inner">
+              <div class="item-title" data-localize="settings.diary.brands">Show Brands</div>
+              <div class="item-after">
+                <label class="toggle toggle-init">
+                  <input type="checkbox" field="diary" name="brands" />
```

Comment:
The input name should be "show-brands" instead of "brands" to better describe what this setting does. It also needs to match the property name in firstTimeSetup (which it currently doesn't).

Reply:
Now I see it, okay, thanks! 

I'll adapt this right away and _try to_ merge this.

Reply:
@EmilJunker : Should be better now, my local testing seems to agree too.